ReSharper.FSharp/src/Daemon.FSharp/src/FSharpSymbolHighlightingUtil.cs

```diff
@@ -82,6 +98,13 @@ public static string GetHighlightingAttributeId([NotNull] this FSharpSymbol symb
         case FSharpEntity entity when !entity.IsUnresolved:
           return GetEntityHighlightingAttributeId(entity);
 
+        case FSharpParameter parameter: // FSharpParameter is a static member in a generic constraint, not a method parameter
+          if (parameter.FullName.Equals("new"))
+            return FSharpHighlightingAttributeIdsModule.Keyword;
+          return parameter.Type.GenericArguments.Count == 2 && parameter.Type.GenericArguments[0].IsUnit
```

Comment:
And the same here. It'd be much easier to reason about this code in future if it well-covered with tests.

Reply:
This code was removed.